frontend/ui/src/pages/Channels/ConnectedChannelsList/ChannelListItem.tsx

```diff
@@ -74,18 +51,19 @@ const ChannelItem = (props: ChannelItemProps) => {
     <>
       <div>
         <div className={styles.channelItem}>
-          <div className={styles.channelLogo}>
-            <ChannelIcon channel={channel} />
+          <div className={styles.channelLogoWrapper}>
+            {channel.metadata?.imageUrl ? <IconChannelFilter channel={channel} /> : <AiryLogoIcon />}
           </div>
-          <div className={styles.channelNameButton}>
-            <div className={styles.channelName}>
-              {channel.metadata.name}
-              {channel.connected && (
-                <div className={styles.connectedHint}>
-                  Connected <CheckMark />
-                </div>
-              )}
+
+          <div className={styles.channelName}>{channel.metadata?.name}</div>
+          {isPhoneNumberSource() && <div className={styles.channelId}>{channel.sourceChannelId}</div>}
```

Comment:
you need to check isPhoneNumberSource first and the return either channelName or channelId, otherwise it will have a gap where the name would have been. This might happen also in sourceInfo!

Reply:
For sms and whatsapp (UI design), we always need to return both channel name and channel id. I fixed the css, so it works for all sources. 
![Screen Shot 2021-03-12 at 11 25 57](https://user-images.githubusercontent.com/59567201/110927437-eb524c80-8325-11eb-9c79-c3cf11e266c5.png)
![Screen Shot 2021-03-12 at 11 26 19](https://user-images.githubusercontent.com/59567201/110927446-edb4a680-8325-11eb-9256-9df5147d496a.png)

Reply:
I am not sure I understand the statement. If the user does not input a name, they don't get a name rendered in the UI. The current design looks like that. If name is not filled in the input bar, then the name filed will be blank, as it is optional
![Screen Shot 2021-03-12 at 11 25 57](https://user-images.githubusercontent.com/59567201/110928119-cca08580-8326-11eb-874a-0903c3e06a64.png)

Reply:
Yes the name is optional but on your screenshot you added always a name to it, if you don't add a name it looks like this:

![Screenshot 2021-03-12 at 11 45 46](https://user-images.githubusercontent.com/49147864/110929711-89dfad00-8328-11eb-8935-3080a273116e.png)